### PR TITLE
migrate MobileDialogPrimitive to Tailwind

### PR DESCRIPTION
**Pull request by DSil**
[FEPLT-1793](https://kiwicom.atlassian.net/browse/FEPLT-1793)

# This Pull Request meets the following criteria:

### For tailwind migrations:

- [ ] ~Visual regression test suite has been added and run before the migration.~ **This component can't be tested in isolation, it is tested in its usages on other components.**
- [ ] ~`tailwind-migration-status.yaml` file has been updated with the migrated component(s).~ **This component is not exported, so it's not tracked on this file**
- [ ] ~Visual regression test suite has been run after the PR approval and no visual changes were identified.~ **Same as above**

Marked as chore because the component is not exported nor documented.
 Storybook: https://orbit-mainframev-tw-mobile-dialog.surge.sh

[FEPLT-1793]: https://kiwicom.atlassian.net/browse/FEPLT-1793?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx [hunk 1]
```diff
@@ -140,21 +48,40 @@ const DialogContent = ({
...
+      />
+      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions */}
+      <div
+        className={cx(
+          "bg-ink-dark shadow-raised-reverse p-md bottom-md inset-x-md fixed z-[10012] box-border max-h-[calc(100%-theme(spacing.xl))] w-[calc(100%-theme(spacing.xl))] overflow-y-scroll rounded-[12px]",
+          "[transition:transform_theme(transitionDuration.normal)_ease-in-out,_visibility_theme(transitionDuration.fast)_linear]",
+          "[-ms-overflow-style:none] [scrollbar-width:none] [&::-webkit-scrollbar]:hidden [&_img]:max-w-full",
+          shown
+            ? "visible translate-y-0 opacity-100"
+            : "duration-normal invisible translate-y-[calc(100%+16px)] opacity-0",
```
Comment (mainframev) [inline packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx:70]:
```suggestion
            : "duration-normal invisible translate-y-[calc(100%+theme(spacing.md))] opacity-0",
```

File: packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx [hunk 2]
```diff
@@ -140,21 +48,40 @@ const DialogContent = ({
...
   return (
-    <StyledDialog role="dialog" id={dialogId} data-test={dataTest}>
-      <StyledDialogOverlay shown={shown} ref={overlay} onClick={handleClickOutside} />
-      <StyledDialogWrapper
-        shown={shown}
+    <div className="w-full" role="dialog" id={dialogId} data-test={dataTest}>
+      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
+      <div
+        className={cx(
+          "fixed inset-0 z-[10011] h-full w-full bg-[rgba(23,27,30,0.6)]",
```
Comment (mainframev) [inline packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx:55]: Non-blocking. Just for discussion. I do not think this z-index value is okay to use, that's too big and should probably just be bigger than the rest of our values or `zIndexOnTheTop` 🤔

**Review by mainframev — CHANGES_REQUESTED** [review]
LGTM 🚀 Just one bug, one suggestion and one question ;)

File: packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx (hunk 1 above)
Comment (oreqizer) [reply packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx:70]: adjuted 👍

File: packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx [hunk 3]
```diff
@@ -140,21 +48,40 @@ const DialogContent = ({
...
+          "[transition:opacity(transitionDuration.normal)_ease-in-out,_visibility_theme(transitionDuration.fast)_linear]",
```
[the remark this answers is not in the document]
Comment (oreqizer) [reply packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx:56]: adjuted 👍

File: packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx (hunk 2 above)
Comment (oreqizer) [reply packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx:55]: those were like this before. we can clean up z-indices later on

Comment (mainframev) [reply packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx:55]: alright 👍🏻

**Review by mainframev — APPROVED** [review]

**Pull request merged**